src/components/user/UserReservations.vue

```diff
@@ -49,14 +58,14 @@ onMounted(async () => {
   </template>
   <template v-else>
     <div v-if="ticketList" class="user-reservations">
-      <BaseCard>
+      <BaseCard width="100%">
```

Comment:
use class instead of inline style

Reply:
this is actually a prop. You commented on this earlier and i have changes how this is handled in the component